task/04-arrays-tasks.js

```diff
@@ -536,7 +625,12 @@ function selectMany(arr, childrenSelector) {
  *   [[[ 1, 2, 3]]], [ 0, 0, 1 ]      => 2        (arr[0][0][1])
  */
 function getElementByIndexes(arr, indexes) {
-    throw new Error('Not implemented');
+   let deep = (array, index) => {
```

Comment:
Why did you create internal separate function for this logic? I think it is an overengineering solution.